src/selectors/fridge.js

```diff
@@ -251,7 +251,7 @@ export const selectSelectedFridgeIsInColdBreach = state => {
 export const selectSelectedFridgeSensorIsLowBattery = state => {
   const fridge = selectSelectedFridge(state);
   const { batteryLevel } = fridge;
-  return batteryLevel <= VACCINE_CONSTANTS.LOW_BATTERY_PERCENTAGE;
+  return batteryLevel && batteryLevel <= VACCINE_CONSTANTS.LOW_BATTERY_PERCENTAGE;
```

Comment:
Had to check to make sure but if this is `0` then this returns `0` (or  `isLowBattery(0)` == `false`).

![image](https://user-images.githubusercontent.com/35858975/108285348-ce759e00-71eb-11eb-82a5-257a135360de.png)